tools/csv-generator/csv-generator.go

```diff
@@ -150,8 +152,15 @@ Performance Addon Operator provides the ability to enable advanced node performa
 				Email: email,
 			})
 		}
+		// Override generator default values
+		if len(userData.Maintainers) == 0 {
+			operatorCSV.Spec.Maintainers = nil
```

Comment:
why the default zero value was not good? What happens now without this change?

Reply:
The new generator creates a default maintainer / email node and will appear in the resulting csv.
The only way to shut it out is to "nil".

Reply:
This is actually something we should use I believe. It's something long due. We just need to figure out the right values.

Reply:
agreed